### PR TITLE
Add release notes for the upcoming testnet.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Add release notes for the upcoming testnet.

### CODE REVIEW

File: source/mainnet/net/resources/release-notes.rst [hunk 1]
```diff
@@ -4,6 +4,22 @@
 Release Notes - Testnet
 =======================
 
+????????????????????
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:7]: What should this be @concordium-cl  @dg-concordium ?

File: source/mainnet/net/resources/release-notes.rst [hunk 2]
```diff
@@ -4,6 +4,22 @@
...
+---------------------
+
+Concordium Node 4.2.1 is a maintenance release bringing performance improvements and bugfixes. The highlights are
+
+- Significant decrease in node startup time. The exact improvements are platform
+  dependent, but startup should be at least 6 times faster on mainnet.
+- Significant decrease in node memory use. On mainnet 4.2.1 node should use less
+  than 50% compared of memory compared to 4.1.1.
+- Reduced CPU use of passive nodes in Windows, Mac, and Linux distributions.
+
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:21]: Please feel free to make it prettier @dg-concordium

**Comment by dg-concordium** [comment]
I actually already had a pull request open with release notes for this. Will copy this to that and close this one.

**Comment by abizjak** [comment]
> I actually already had a pull request open with release notes for this. Will copy this to that and close this one.

Ok. Sorry about that, did not know. Thanks.

**Pull request closed without merging**